Practice1/src/css/header.css

```diff
@@ -0,0 +1,169 @@
+.page-header {
+  position: relative;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 63px;
+  padding: 25px 0px 0px 51px;
+  width: calc(100% - 92px);
+}
+
+.nav-img {
+  position: absolute;
+  z-index: -1;
+  width: 100%;
+  height: 100%;
+}
+
+.nav .btn-tag {
+  display: flex;
+  color: var(--white);
+  align-items: center;
+}
+
+.nav .btn-tag .btn-login {
+  padding-right: 20px;
+}
+
+.nav .btn-tag .btn-lyp {
+  border: 1px solid var(--white);
+  padding: 10px 10px;
+  width: 120px;
+}
+
+.nav .logo {
+  color: var(--black);
+  display: flex;
+  font-size: 10px;
+  align-items: center;
+}
+
+.nav .nav-input {
+  font-size: 20px;
+  color: var(--white);
+  padding-left: 10px;
+}
+
+.search {
+  display: flex;
+  height: 100px;
+  width: calc(100% - 115px);
+  justify-content: center;
+  margin: 0px auto;
+  margin-top: 306px;
+  margin-bottom: 100px;
+}
+
+.area {
+  margin-top: 15px;
+  position: relative;
+  height: 20px;
+}
+
+.area .area-input {
+  padding: 5px 0px 5px 35px;
+  height: 40px;
+}
+
+.area .icon {
+  position: absolute;
+  height: 20px;
+  left: 10px;
+  top: 13px;
+}
+
+.hospitals {
+  margin-top: 15px;
+  position: relative;
+  height: 20px;
+}
+
+.hospitals .hospitals-input {
+  padding: 5px 0px 5px 35px;
+  outline: none;
+  width: 400px;
+  height: 40px;
+}
+
+.hospitals .icon {
+  position: absolute;
+  height: 15px;
+  left: 10px;
+  top: 17px;
+}
+
+.insurance {
+  margin-top: 15px;
+  position: relative;
+  height: 20px;
+}
+
+.insurance .insurance-input {
+  padding: 5px 0px 5px 35px;
+  outline: none;
+  width: 340px;
+  height: 40px;
+}
+
+.insurance .icon {
+  position: absolute;
+  height: 15px;
+  left: 10px;
+  top: 17px;
+}
+
+.example .example-button {
+  width: 100%;
+  height: 55px;
+  margin-top: 15px;
+  padding: 20px;
+  background: var(--blue-dark);
+  color: var(--white);
+  border: 1px solid var(--gray);
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+	font-size: 36px;
```

Comment:
Code style here

Reply:
Fixed

Reply:
@duylong13 Pls reload the Github page and check this file once again.